### PR TITLE
fix: Remove index from user prompt messages and standardize

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/37308523/CHI-2177

### CODE REVIEW

**Comment by vbcypherock** [comment]
Task linked: [CHI-2177 Revert the prompt messages](https://app.clickup.com/t/37308523/CHI-2177)

File: src/controller_main.c [hunk 1]
```diff
@@ -630,38 +599,26 @@ void desktop_listener_task(lv_task_t* data)
...
                     } else {
                         flow_level.level_two = LEVEL_THREE_RECEIVE_TRANSACTION;
-                        snprintf(flow_level.confirmation_screen_text, sizeof(flow_level.confirmation_screen_text),
-                                 UI_TEXT_RECEIVE_PROMPT,
-                                 get_coin_name(coin_index, receive_transaction_data.network_chain_id),
-                                 account_tag, wallet.wallet_name);
                     }
+
+                    snprintf(flow_level.confirmation_screen_text, sizeof(flow_level.confirmation_screen_text),
+                             UI_TEXT_RECEIVE_PROMPT, token_symbol, chain_name, wallet.wallet_name);
```
Comment (amanCypherock) [inline src/controller_main.c:621]: Do we need to have Coin symbol and name in case of Bitcoin and it's forks too?

Comment (amanCypherock) [reply src/controller_main.c:621]: For Chains that do not support multiple tokens/denominations, we can leave the token/denomination name for send and receive txns confirmations.

Comment (ujjwal-cyph) [reply src/controller_main.c]: changed

**Review by amanCypherock — APPROVED** [review]

**Pull request merged**